### ISSUE TITLE
Memory leaks

### ISSUE THREAD

**Issue by Zetaphor** [webaverse/metachromium-bin#16]
I've been having consistent issues with memory leaks from console errors. The result ranges anywhere from all Chromium instances crashing up to Windows itself having a meltdown

This is hard to reproduce. Anecdotal evidence says the problem gets worse with multiple refreshes of the same tab. 

Sometimes I can throw a reference error in the render loop and I'm able to close the tab before things get out of control, other times that will happen and MC will crash, and yet other times it takes down my OS. Once I clear out all of the MC processes and restart, it doesn't happen again for a while

**Comment by avaer** [comment]
Were you able to reproduce this without any refreshes?

E.g. start MetaChromium, trigger the leak condition, and it still breaks?

**Comment by Zetaphor** [comment]
No, it appears to happen after a number of refreshes. Killing all active or rogue Chromium instances and restarting resolves the issue temporarily

**Comment by avaer** [comment]
That's useful info, thanks. I suspect in this case it would be overlay layers that do not get properly collected. Currently we do not tear down the sessions on page reload, so this might be leaking overlay textures.
